File: codewars/8kyu/sort-and-star/old_solution.go
// https://www.codewars.com/kata/57cfdf34902f6ba3d300001e
// 8 kyu
// Sort and Star

package kata

import (
	"sort"
	"strings"
)

// My Solution
func TwoSort(arr []string) string {
	result := arr[0]

	for i := 1; i < len(arr); i++ {
		word := arr[i]
		if strings.Compare(result, word) == 1 {
			result = word
		}
	}
	return strings.Join(strings.Split(result, ""), "***")
}

// Codewars Solution
func TwoSort2(arr []string) string {
	sort.Strings(arr)
	return strings.Join(strings.Split(arr[0], ""), "***")
}

File: codewars/6kyu/create-phone-number/old_solution.py
# https://www.codewars.com/kata/525f50e3b73515a6db000b83
# 6 Kyu
# Create Phone Number

# My Solution
def func(x):
    y = "".join(str(ex) for ex in x)
    return f"({y[:3]}) {y[3:6]}-{y[6:]}"


a = func([1, 2, 3, 4, 5, 6, 7, 8, 9, 0])
a
